core/modules/taxonomy/tests/taxonomy.test

```diff
@@ -549,14 +543,6 @@ class TaxonomyTermTestCase extends TaxonomyWebTestCase {
     // free-tagging field created by the default profile.
     $edit[$instance['field_name'] . "[$langcode]"] = backdrop_implode_tags($terms);
 
-    // Preview and verify the terms appear but are not created.
-    $this->backdropPost('node/add/page', $edit, t('Preview'));
-    foreach ($terms as $term) {
-      $this->assertText($term, 'The term appears on the node preview.');
-    }
-    $tree = taxonomy_get_tree($this->vocabulary->machine_name);
-    $this->assertTrue(empty($tree), 'The terms are not created on preview.');
-
```

Comment:
I'm not sure I understand.  You want new tests that add terms to a new node, then upload an image, and then check that the terms have not been created? I'm not sure I see the value in that.